Formula/libsigrokdecode.rb

```diff
@@ -2,18 +2,16 @@ class Libsigrokdecode < Formula
   desc "Drivers for logic analyzers and other supported devices"
   homepage "https://sigrok.org/"
   license "GPL-3.0-or-later"
-
   head "git://sigrok.org/libsigrokdecode", branch: "master"
 
   stable do
```

Comment:
Unless I'm missing something, a `stable` block isn't required here, is it?

Reply:
Good catch! This formula was probably adapted from `libsigrok` when it was introduced in #86791. `libsigrok` uses a `stable` block because it also has `resource` block but the same isn't true here. I've updated this accordingly.